CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/Habit.java

```diff
@@ -65,6 +67,15 @@ public void updateHabitEvent(int position, HabitEvent updatedHabitEvent) {
         this.habitEventTable.set(position, updatedHabitEvent);
     }
 
+    public Boolean getPublic() {
+        return isPublic;
+    }
+
+    public void setPublic(Boolean aPublic) {
```

Comment:
I'd recommend naming the input bool "isPublic" for clarity. You can then use this.isPublic = isPublic;
(this avoids ambiguities with the variable names)

Reply:
Also can this be moved down & arranged with the other getters/setters?